ALUM/ALUM.xcodeproj/project.pbxproj

```diff
@@ -696,6 +670,24 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		074631412A46C5AC0003E07D /* Setup Firebase Environment GoogleService-Info.plist */ = {
```

Comment:
This sets up a build script which copies over the relevant GoogleService-Info-*.plist file to "GoogleService-Info.plist" which is used for compiling the application. 

When "Archive" is clicked in xcode, "CONFIGURATION" is set to "Release" so the prod googleservice file is used. Otherwise, dev googleservice file is used. These secrets are found in the google drive (check README)